common/src/main/java/org/dash/wallet/common/util/GenericUtils.kt

```diff
@@ -45,7 +54,7 @@ object GenericUtils {
     }
 
     fun getDeviceLocale(): Locale {
-        val countryCode = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+        val countryCode = if (!isRunningUnitTest) {
             LocaleList.getDefault()[0].country
         } else {
             Locale.getDefault().country
```

Comment:
While we don't need the `Build.VERSION_CODES.N` check anymore, the unit test fails on `LocaleList.getDefault()[0]`, and mocking the static method doesn't seem to work. This workaround is somewhat ugly, but will do for now.